src/test/java/hng_java_boilerplate/notificationSettings/service/NotificationSettingsServiceTest.java

```diff
@@ -0,0 +1,64 @@
+package hng_java_boilerplate.notificationSettings.service;
+
+import hng_java_boilerplate.notificationSettings.entity.NotificationSettings;
+import hng_java_boilerplate.notificationSettings.repository.NotificationSettingsRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import static org.mockito.Mockito.*;
+
+
+import java.util.Optional;
+import java.util.UUID;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class NotificationSettingsServiceTest {
```

Comment:
Include e2e tests

Reply:
done

Reply:
Still only unit tests here

Reply:
what do i do, ive made the changes and pushed... i didnt see any prompt to fetch changes. what do i do?